configure.ac

```diff
@@ -1142,6 +1142,24 @@ if test x"$with_png" != "xno"; then
   )
 fi
 
+# look for libimagequant with pkg-config (only if libpng is enabled)
+AC_ARG_WITH([imagequant],
+  AS_HELP_STRING([--without-imagequant], [build without imagequant (default: test)]))
+
+if test x"$with_imagequant" != "xno" && test x"$with_png" != "xno"; then
```

Comment:
You conventionally write `x"no"` to mirror the left-hand side of the `!=`. Though of course it makes no practical difference.

Reply:
Yeah, it’s copied over from the existing png test a few lines above.